### DISCUSSION TITLE
[Bug report] vercel.com by default redirects me to a team I'm not longer part of

### DISCUSSION THREAD

**Discussion by JorensM** [vercel/community#5700] [Help] [answered]
### Summary

Hello, I don't know where else to submit this bug report. I tried Vercel support but I need be on a paid plan to report a bug for dashboard related issues.

So basically when I open Vercel with the url `vercel.com`, I get redirected to a team. The thing is, I'm not longer a part of the team so I basically get redirected to a 404 page. So I thought maybe add an option for `Default team` in the settings that will show up when you first open Vercel. Or at least redirect me to a team that I'm still a part of

Thank you.

### Steps to Reproduce

1. Create a Vercel account and be added as a team member to a team.
2. Create another team on your own Vercel account
3. Be removed from the initial team
4. Go to `vercel.com`
5. See that you're being redirected to the team you're no longer part of.

**Comment by swarnava** [comment]
Hi,

Are you login using `j***********u@gmail.com` email? Can you record a screencast and share with us?

**Reply by JorensM** [reply]
As requested, here is a screen recording demonstrating the bug

https://github.com/vercel/community/assets/17122123/8db96351-2227-4c56-8ace-1671dd073a51

You can see in the URL that the page gets redirected to a team I'm no longer part of ('fankee' is the ID of the team).

**Reply by swarnava** [reply, marked as answer]
Are you able to reproduce if you visit https://vercel.com/dashboard and then visit vercel.com? Can you also try logging out and then login?

**Reply by JorensM** [reply]
Re-logging fixed this, thanks!
